pom.xml

```diff
@@ -680,7 +680,6 @@
                 presence of this file.
               -->
               <xmlOutput>true</xmlOutput>
-              <spotbugsXmlOutput>false</spotbugsXmlOutput>
```

Comment:
Based on my local tests, deleting this entire `<configuration>` block still gives us the result we want (generating a `spotbugsXml.xml` file for Jenkins), so I think we should just delete the whole block: the less code there is to maintain, the better. The comment about `excludeFilterFile` is valuable, but it could just be moved to below `<phase>verify</phase>`.

Also the same change needs to be made to `jenkinsci/pom` for consistency.